### DISCUSSION TITLE
pk sim ver 10

### DISCUSSION THREAD

**Discussion by amaghrabi** [Open-Systems-Pharmacology/Forum#1041] [Questions & Problems] [answered]
hi everyone

they said we can get a predefined pbpk model individuals for different stages of chronic kidney disease (CKD) in version 10 of pk sim 

i downloaded it, and searched in the templated available in individual and its showing that  no template available 

please if someone how to fix my issue 

thank

**Comment by StephanSchaller** [comment, marked as answer]
Dear @amaghrabi , please look in the population templates database as described here:
https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v10.0
